pubspec.yaml

```diff
@@ -35,12 +35,12 @@ dependencies:
 dev_dependencies:
   analyzer: ^4.7.0
   archive: ^3.1.2
-  cli_pkg: ^2.4.2
+  cli_pkg: ^2.4.3
```

Comment:
This update is causing CI to fail on Node v14. I submitted an [issue](https://github.com/google/dart_cli_pkg/issues/124) and [PR to fix it](https://github.com/google/dart_cli_pkg/pull/125).

Reply:
Fixed in https://github.com/oddbird/dart-sass/commit/38f1f5ffdd0f42131692ca78a120636a72b3ecaa